lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
```

Comment:
The above does work in the console:
```
> edx.HtmlUtils.interpolateHtml(edx.HtmlUtils.HTML('<p>{text}</p>'), {'text': "I'm escaped!"})
HtmlSnippet {text: "<p>I&#x27;m escaped!</p>"}
```
@andy-armstrong: You mentioned that you didn't like using interpolateHtml() where the first argument is HTML, but it looks better than joinHtml() if you look at the two example above.  Do you agree, or have an alternative?

@kevinjkim: When either of these work, then `accountSettingsLink` should be an HTML Snippet, and interpolateHtml() should respect it below.  You would need to dig deeper or provide more details regarding how it doesn't work.